qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -134,14 +134,18 @@ def get_file_md5sum(file_name, session, timeout):
                         error_context.context("Transferring data from %s to %s" %
                                               (vm_src.name, vm_dst.name),
                                               test.log.info)
+                        if params.get("using_guest_interface") == "yes":
```

Comment:
Can use `get_boolean`

Reply:
Updated, please help review it again, thanks a lot.